src/main/resources/mybatis/collections.xml

```diff
@@ -102,4 +102,48 @@
         ) as collection
     </select>
 
+
+    <select id="getStatisticalTimeSeriesJson" parameterType="String" resultType="String">
+        select row_to_json(fsts)
+          from ( select 'Feature' as "type",
+                          ml.monitoring_location_identifier as "id",
+                          json_build_object(
+                             'type', 'Point',
+                             'coordinates', json_build_array(
+                                 to_json(trunc(st_x(st_transform(geom, 4326))::numeric, 7)),
+                                 to_json(trunc(st_y(st_transform(geom, 4326))::numeric, 7))
+                             )
+                          ) as "geometry",
+                          json_build_object('samplingFeatureName', site_name) as "properties",
+                          ( select array_to_json(array_agg(sts))
+                              from ( select distinct 
```

Comment:
This operation coded in this way could have very poor performance. It may instantiate the entirety of the distinct data before filtering by monitoring_location_id.

Reply:
We will optimize as performance proves itself. Agile methods say not to preoptimise. That said, I would suggest that the values in the distinct query be in a table and not in every value. When we know more about the structure then we can certainly revise the query.

Reply:
I do not recall Agile saying be stupid and intentionally write bad code.

Reply:
Foolish comment. This is not bad code. it is working withing the database as it is. Stop being confrontational for the sake of being so.